tensorboard/writer/event_file_writer_test.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+# """Integration tests for the Writer."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+# tf.compat.v1.disable_v2_behavior()
+
+import glob
+import os
+import tensorflow as tf
```

Comment:
Rather than depending on actual tensorflow, for tf.test.TestCase we can use the test case @orionr was adding in the other PR once that's ready, and instead of `tf.train.summary_iterator()` we could just use the `PyRecordReader_New` implementation from the TF stub.

That way this test can still run in no-TF mode.

Reply:
Work in progress on that new TestCase class over in https://github.com/tensorflow/tensorboard/pull/2089

Reply:
And this has landed, so feel free to do 

```
from tensorboard import test as tb_test
```